cypress/support/cypress-support/src/main.js

```diff
@@ -421,6 +429,9 @@ export default function (module) {
 
       if (!UTILS.getEnvVariable(CONSTANTS.DEVICE_MAC)) {
         cy.log(CONSTANTS.DEVICE_MAC_UNAVAILABLE).then(() => {
+          logger.error(
```

Comment:
Let's set this as the assert failure message so the user will more easily see it.